packages/astro/src/core/server-islands/endpoint.ts

```diff
@@ -74,7 +75,11 @@ export function createEndpoint(manifest: SSRManifest) {
 			});
 		}
 
-		const props = data.props;
+		const key = await manifest.key;
+		const encryptedProps = data.props;
```

Comment:
Let's rename `props` to `encriptedProps` at this point so it's more visible to us.